doc/examples/segmentation/plot_regionprops.py

```diff
@@ -22,6 +30,9 @@
 
 image = rotate(image, angle=15, order=0)
 
+rrr, ccc = ellipse(100, 100, 60, 50)
```

Comment:
```suggestion
rr, cc = ellipse(100, 100, 60, 50)
```

Reply:
Ok. Need to change the next line as well (hence committing manually 3094978).